### PR TITLE
test

### PR DESCRIPTION
**Pull request by Orlyga01**

### CODE REVIEW

File: go_to_office/lib/admin/SeatsList.dart [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        appBar: AppBar(
+          title: Text("Seats"),
+        ),
+        body: SingleChildScrollView(
+          padding: EdgeInsets.all(15.0),
+          child: Column (
+            mainAxisSize: MainAxisSize.min,
+             children: <Widget>[
+                 ElevatedButton(
+                child: Text('Add New Seat'),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/SeatsList.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/SeatsList.dart [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+class _SeatsListPageState extends State<SeatsListPage> {
+    final String officeId;
+     _SeatsListPageState(this.officeId);
+  @override
+  Widget build(BuildContext context) {
+
+    // Object officateAssreddress = {};
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("Seats"),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/SeatsList.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/SeatsList.dart [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+class _SeatsListPageState extends State<SeatsListPage> {
+    final String officeId;
+     _SeatsListPageState(this.officeId);
+  @override
+  Widget build(BuildContext context) {
+
+    // Object officateAssreddress = {};
+    return Scaffold(
```
Comment (OzBenDev) [inline go_to_office/lib/admin/SeatsList.dart:32]: warp it with  SingleChildScrollView (see login_page.dart)

File: go_to_office/lib/admin/office.page.dart [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  } else {
+    this.officeName = '';
+    this.officeCoutnry = '';
+    this.officeDescription = '';
+  }
+
+    // Object officateAssreddress = {};
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("Office Manager"),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/office.page.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/office.page.dart [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+                        InputDecoration(hintText: 'Please insert Country'),
+                    onChanged: (String inputString) {
+                      setState(() {
+                        officeCoutnry = inputString;
+                      });
+                    }),
+                TextFormField(
+                  initialValue: this.officeDescription,
+                    decoration: InputDecoration(
+                        hintText: 'Please insert Office Description'),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/office.page.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/office.page.dart [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+                        InputDecoration(hintText: 'Please insert Office name'),
+                    onChanged: (String inputString) {
+                      setState(() {
+                        officeName = inputString;
+                      });
+                    }),
+                TextFormField(
+                  initialValue: this.officeCoutnry,
+                    decoration:
+                        InputDecoration(hintText: 'Please insert Country'),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/office.page.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/office.page.dart [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+          backgroundColor: Colors.blueAccent,
+        ),
+        body: SingleChildScrollView(
+            padding: EdgeInsets.all(15.0),
+            child: Column(
+              children: <Widget>[
+                TextFormField(
+                  initialValue: this.officeName,
+                    decoration:
+                        InputDecoration(hintText: 'Please insert Office name'),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/office.page.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/office.page.dart [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+                  initialValue: this.officeDescription,
+                    decoration: InputDecoration(
+                        hintText: 'Please insert Office Description'),
+                    onChanged: (String inputString) {
+                      setState(() {
+                        officeDescription = inputString;
+                      });
+                    }),
+                    ElevatedButton(
+                      child: Text('Add New Seat'),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/office.page.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/officesList.dart [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+        appBar: AppBar(
+          title: Text("Offices"),
+        ),
+        body: SingleChildScrollView(
+          padding: EdgeInsets.all(15.0),
+          child: Column (
+            mainAxisSize: MainAxisSize.min,
+             children: <Widget>[
+              ElevatedButton(
+                child: Text('Add New Office'),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/officesList.dart]: Add to the Strings and import it

File: go_to_office/lib/admin/seat.page.dart [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+          backgroundColor: Colors.blueAccent,
+        ),
+        body: SingleChildScrollView(
+            padding: EdgeInsets.all(15.0),
+            child: Column(
+              children: <Widget>[
+                  TextFormField(
+                  initialValue: this.seatLocation,
+                    decoration: InputDecoration(
+                        hintText: 'Please insert Seat Loaction in the room'),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/seat.page.dart:48]: Add to the Strings and import it

File: go_to_office/lib/admin/seat.page.dart [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+  final String seatId;
+  final String officeId;
+  String roomId ;
+  String seatLocation ;
+   _SeatPageState(this.seatId, this.officeId);
+  @override
+  Widget build(BuildContext context) {
+    this.roomId = '1';
+    if (this.seatId != null) {  // call API to get get office also get the office name
+    this.seatLocation = 'Near a window';
```
Comment (OzBenDev) [inline go_to_office/lib/admin/seat.page.dart:30]: Add to the Strings and import it

File: go_to_office/lib/login_page.dart [hunk 12]
```diff
@@ -32,13 +33,13 @@ class _LoginPageState extends State<LoginPage> {
   Widget build(BuildContext context) {
     void showSnack(String text) {
       final snackBar = SnackBar(content: Text(text));
```
Comment (OzBenDev) [inline go_to_office/lib/login_page.dart]: remove this line also - make this method empty for now

File: go_to_office/lib/login_page.dart [hunk 13]
```diff
@@ -32,13 +33,13 @@ class _LoginPageState extends State<LoginPage> {
...
-      ScaffoldMessenger.of(context).showSnackBar(snackBar);
+     // ScaffoldMessenger.of(context).showSnackBar(snackBar);
     }
 
     void onError(String text) {
       final snackBar =
           SnackBar(content: Text(text), backgroundColor: Colors.red);
```
Comment (OzBenDev) [inline go_to_office/lib/login_page.dart]: remove this line also - make this method empty for now

File: go_to_office/lib/admin/officesList.dart [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+                shrinkWrap: true,
+                padding: const EdgeInsets.all(8),
+                itemCount: officesList.length,
+                itemBuilder: (BuildContext context, int index) {
+                  return new GestureDetector(
+                    onTap: () => openOfficePage(officesList[index].id),
+                    child: Container(
+                      height: 50,
+                      margin: EdgeInsets.all(2),
+                      child: Text('${officesList[index].name}',
```
Comment (OzBenDev) [inline go_to_office/lib/admin/officesList.dart:57]: null check? can it be problematic?

File: go_to_office/lib/admin/officesList.dart [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+                },
+              ),
+              ListView.builder(
+                scrollDirection: Axis.vertical,
+                shrinkWrap: true,
+                padding: const EdgeInsets.all(8),
+                itemCount: officesList.length,
+                itemBuilder: (BuildContext context, int index) {
+                  return new GestureDetector(
+                    onTap: () => openOfficePage(officesList[index].id),
```
Comment (OzBenDev) [inline go_to_office/lib/admin/officesList.dart:53]: null check? can it be problematic?

File: go_to_office/lib/admin/SeatsList.dart [hunk 16]
```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'seat.page.dart';
+import '../strings.dart';
+
+void main(officeId) {
+  runApp(new MaterialApp(
+    title: "Seats",
```
Comment (OzBenDev) [inline go_to_office/lib/admin/SeatsList.dart]: Add to the Strings and import it

**Review by OzBenDev — CHANGES_REQUESTED** [review]
Add null checks if needed, use the modify and import the Strings resource file

File: go_to_office/lib/admin/officesList.dart (hunk 15 above)
Comment (Orlyga01) [reply go_to_office/lib/admin/officesList.dart:53]: If the officesList is empty the loop will not run

**Pull request merged**